projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/monitoring/DataJobMetricsTest.java

```diff
@@ -273,4 +273,40 @@ void testIncrementWatchTaskInvocations() {
         meterRegistry.counter(DataJobMetrics.TAURUS_DATAJOB_WATCH_TASK_INVOCATIONS_COUNTER_NAME);
     Assertions.assertEquals(5.0, counter.count(), 0.001);
   }
+
+  @Test
+  @Order(14)
```

Comment:
why do you care about the order?

Reply:
Overall the whole test is miswritten and needs to be rewritten. As far as I remember some of the steps have unnecessary dependencies between them.

Reply:
So let's keep the order for now.